testcases/OpTestFastReboot.py

```diff
@@ -100,9 +100,11 @@ def runTest(self):
         try:
             fast_reboot_state = ''.join(c.run_command(
                 "cat /proc/device-tree/ibm,opal/fast-reboot"))
+            # skipTest throws exception when passing null at end
+            # so remove the devicetree string passed
+            # exception from skipTest produces malformed XML
             if fast_reboot_state[:4] != "okay":
-                self.skipTest("Fast reboot not supported: {}".format(
-                    fast_reboot_state))
```

Comment:
Alternatively, we could skip the NULL by doing `fast_reboot_state[:-1]` which will trim off the last character and thus be okay.

Reply:
> Alternatively

Good idea, fixed.